Kinvey/Kinvey/Geolocation.swift

```diff
@@ -108,14 +108,6 @@ public func <- (left: inout GeoPoint?, right: (String, Map)) {
     left <- (map, transform)
 }
 
-/// Override operator used during the `propertyMapping(_:)` method.
-public func <- (left: inout GeoPoint!, right: (String, Map)) {
```

Comment:
@heyzooi Can you explain a little bit what this function is for, why it was removed, and will it break anything?

Reply:
the compiler was warning/complaining the `Geopoint!` unnecessary overloads the `Geopoint?` version, which make sense, but running the unit tests the compiler is not smart enough to rewire the call for the optional version of the method. So I guess we will need to keep this warning for now since it seems to be a compiler bug. I will confirm that once again to make sure and if so I will create a ticket for the Swift compiler team.